rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp

```diff
@@ -818,6 +818,8 @@ rmf_fleet_msgs::msg::RobotState convert_state(const TaskManager& mgr)
         .x(p.x())
         .y(p.y())
         .yaw(l.orientation())
+        .obey_approach_speed_limit(false)
```

Comment:
Maybe we can create a ticket to correctly populate these fields for future work.

Reply:
Done in #143